configure.ac

```diff
@@ -241,24 +241,24 @@ AC_SEARCH_LIBS([clock_gettime], [rt],
 dnl check for libraries
 
 OMPI_CHECK_PORTALS4([portals4],
-    [transport="portals4"
-     transport_portals4="yes"],
+    [transport_portals4="yes"],
     [transport_portals4="no"])
 
 OPAL_CHECK_OFI([ofi],
-    [transport="ofi"
-     transport_ofi="yes"],
+    [transport_ofi="yes"],
     [transport_ofi="no"])
 
 # assure that only ofi is enabled if portals4 is also detected
-if test -n "$with_ofi" ; then
+if test "$transport_ofi" = "yes" ; then
```

Comment:
I think we actually want ```$with_ofi``` here to check if the user asked for it.  With the new logic, an auto detected OFI library will prevent the user from creating a Portals build.